### ISSUE TITLE
docComments produces false positive for TODO comments when ahead of function signatures

### ISSUE THREAD

**Issue by rogerluan** [nicklockwood/SwiftFormat#1457]
By enabling `docComments`, it transforms code like this:

```diff
-// TODO: Clean up this mess
+/// TODO: Clean up this mess
func doSomething() {
    …
}
```

Probably because it precedes a function signature, so it thinks it should be marked as a doc comment instead of a simple comment.

**Comment by nicklockwood** [comment]
Yeah, it shouldn't do that - thanks for flagging it

**Comment by nicklockwood** [comment]
@rogerluan fixed in 0.51.11

**Comment by rogerluan** [comment]
Awesome! Thank you!
